mylist_class.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'enum_module'
+
+class MyList
+  include Enumerable
+
+  def initialize(*args)
+    @list = args
+  end
```

Comment:
- [Optional] Dear @FilbertKimasa, assigning an object/array this way is not a professional way of doing it since object/array shares the same memory location. What I would suggest doing is with a `.each` with a separate `push` action. Something like `args.each { |arg| @list << arg }`

Reply:
Good reminder @TadesseDev